# Introduction
Below are some tutorials and helfpul links for bioinformatics and data analysis basics. Most information below is related to bulk RNA-sequencing data processing and analysis.
This will be expanded to include single-cell RNA-seq analysis as well. 

## In-house tutorials/notes

| Link | Description |
| ----------- | ----------- |
| [Synapse Quantification & Stats Tutorial](/SynapseQuant_Statistics.html) | Basic R tutorial for graphing and statistics |
| [RNAseq Pipeline w/ Salmon + DESeq2](/matt_aging_astrocyte_tutorial.html) | RNA-seq analysis in R after Salmon quasi-mapping |
|[PDF: RNAseq Data Processing STAR & FeatureCounts](/RNAseq_STARFeatureCounts_Tau12mRibotagISH.pdf) | Command line notes/scripts for RNA-seq data processing with STAR and FeatureCounts using ISH data |
|[HTML: RNAseq Data Processing STAR & FeatureCounts](/RNAseq_STARFeatureCounts_Tau12mRibotagISH.html) | HTML version of the above notes | 

## Below are some additional resources that I've found particularly useful

### Linux Command Line

[Cheatsheet 1](https://phoenixnap.com/kb/linux-commands-cheat-sheet#linux-commands-cheat-sheet-pdf)

[Cheatsheet 2](https://www.guru99.com/linux-commands-cheat-sheet.html)

### Salmon for RNA-seq

[Official Salmon Docs](https://salmon.readthedocs.io/en/latest/salmon.html)

[Example workflow with good explanations](https://hbctraining.github.io/Intro-to-rnaseq-hpc-O2/lessons/08_salmon.html)
 
### General RNA-seq pipelines

[Comparison of methods](https://hbctraining.github.io/Training-modules/planning_successful_rnaseq/slides/RNAseq-strategies_mm.pdf)

### Markdown
https://www.markdownguide.org/basic-syntax/#blockquotes-1

####
Reach out to Jillybeth if you have suggestions, questions or comments related to the information provided above.

